### ISSUE TITLE
Set up persistent logging

### ISSUE THREAD

**Issue by jeffbaumes** [microbiomedata/nmdc-server#766]

**Comment by shreddd** [comment]
Tasks
- [ ] Set up logging directory on CFS (or NFS?)
  - [ ] Decide on naming and directory structure
- [ ] Add config to web services to use persistent logging dir / persistent files 
- [ ] Update Rancher config to mount / use persistent logging dirs

**Comment by mcovalt** [comment]
Yes, that matches how I had expected it to work. CFS for the directory ought to be fine.

**Comment by aclum** [comment]
@shreddd  we need a new assignee here.
